site/app/models/gradeable/Gradeable.php

```diff
@@ -52,14 +52,14 @@
  * @method int getTeamSizeMax()
  * @method \DateTime getTeamLockDate()
  * @method bool isTaGrading()
- * @method bool isScannedExam()
- * @method void setScannedExam($scanned_exam)
  * @method bool isStudentView()
  * @method void setStudentView($can_student_view)
  * @method bool isStudentViewAfterGrades()
  * @method void setStudentViewAfterGrades($can_student_view_after_grades)
  * @method bool isStudentSubmit()
  * @method void setStudentSubmit($can_student_submit)
+ * @method bool canStudentDownload()
```

Comment:
I think you would have to manually define this function instead of using the magic getter/setter stuff - or either add it as a special prefix to look out for 

https://github.com/Submitty/Submitty/blob/master/site/app/models/AbstractModel.php#L130